server/cluster_info.go

```diff
@@ -367,9 +367,19 @@ func (c *clusterInfo) GetRegionStores(region *core.RegionInfo) []*core.StoreInfo
 }
 
 func (c *clusterInfo) getRegionStoresLocked(region *core.RegionInfo) []*core.StoreInfo {
-	var stores []*core.StoreInfo
-	for id := range region.GetStoreIds() {
-		if store := c.core.Stores.GetStore(id); store != nil {
+	stores := make([]*core.StoreInfo, 0, len(region.GetPeers()))
+	for _, p := range region.GetPeers() {
```

Comment:
What is the difference between `region.GetStoreIds` and `region.GetPeers` to get the `storeid` ？

Reply:
`GetStoreIds` will make a map, the `GetPeers` do not alloc new memory.